opentelemetry-sdk/src/opentelemetry/sdk/context/threadlocal_context.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import threading
+
+from opentelemetry.context import Context, RuntimeContext
+
+_CONTEXT_KEY = "current_context"
+
+
+class ThreadLocalRuntimeContext(RuntimeContext):
+    """An implementation of the RuntimeContext interface
+    which uses thread-local storage under the hood. This
+    implementation is available for usage with Python 3.4.
+    """
+
+    def __init__(self) -> None:
+        self._current_context = threading.local()
+
```

Comment:
Can we just initialize this?

```suggestion
    self._current_context._CONTEXT_KEY = Context()
```